lib/sproxyd.js

```diff
@@ -98,7 +97,9 @@ class SproxydClient {
      * @return {undefined}
      */
     setupLogging(logApi) {
-        this.logging = new (logApi || werelogs).Logger('SproxydClient');
+        this.logging = new (logApi || werelogs).Logger('SproxydClient', {
+
```

Comment:
is the space here on purpose?

Reply:
I was going to configure log levels here.